cmd/retester/testdata/testconfig/config.yaml

```diff
@@ -0,0 +1,16 @@
+retester:
+  enabled: true
```

Comment:
Would this enable retester on every repo?

Reply:
No it's enable the option to use retester and `enable: false` just disables the entire retester.

Reply:
In my mind, our configuration at the starting point will be 

```yaml
retester:
  max_retests_for_sha_and_base: 3
  max_retests_for_sha: 9
```

and we add the orgs and the repos on which the retester is enabled.

So `enabled: false` on the global level and enable if needed.

Reply:
It is actually working like that just `enabled: true` didn't enabled not configured orgs. I will changed it in next commit. So `enabled: true` will enable not configured orgs and repos.